dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -1266,21 +1318,34 @@ synchronized String evaluateInterval() {
                 cpuHigh = true;
             }
 
+            boolean lowTput = false;
+            if (throughput / poolSize < lowTputThreadsRatio && queueDepth == 0) {
```

Comment:
```suggestion
            if (queueDepth == 0 && throughput / poolSize < lowTputThreadsRatio) {
```
It makes sense to do the cheaper conditional first.  No reason to do complicated division if queueDepth is not 0.

Reply:
Agreed, changed.